### PR TITLE
change relative layout in fragment traning to constraint layout

### PR DESCRIPTION
**Pull request by ghost**
## change relative layout in fragment training to constraint layout #149 

Converted Relative layout to Constraint Layout in the training fragment .

### CODE REVIEW

File: app/src/main/res/layout/fragment_training.xml [hunk 1]
```diff
@@ -152,13 +152,14 @@
                 android:layout_height="match_parent" />
 
         </LinearLayout>
+        <ProgressBar
+            android:id="@+id/progressBar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true" />
```
Comment (ujjwalagrawal17) [inline app/src/main/res/layout/fragment_training.xml:163]: This attribute won't work in constraint layout to keep the progress bar in center. You need to use constraintStartToStartOf=parent
constraintEndToEndOf=parent
constraintTopToTopOf=parent
,...

**Review by ujjwalagrawal17 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/fragment_training.xml (hunk 1 above)
Comment (ankitadhiman003) [reply app/src/main/res/layout/fragment_training.xml:162]: I added those attributes you mentioned @ujjwalagrawal17, please review.

**Comment by ghost** [comment]
I added those attributes you mentioned @ujjwalagrawal17, kindly review.

File: app/src/main/res/layout/fragment_training.xml (hunk 1 above)
Comment (ujjwalagrawal17) [reply app/src/main/res/layout/fragment_training.xml:163]: Bottom to bottom of attribute is missing.

**Review by ujjwalagrawal17 — CHANGES_REQUESTED** [review]
Please add bottom to bottom of attribute too

**Comment by ghost** [comment]
Added bottom to bottom of attribute @ujjwalagrawal17, please review and let me know if there is any change required.

**Comment by ujjwalagrawal17** [comment]
Thanks @ankitadhiman003 . Congrats on your first PR! Feel free to play around the app and explore the website dashboard.wikiedu.org, www.wikiedu.org. for getting more information about the project.

**Pull request merged**